app/scripts/models/auth_brokers/base.js

```diff
@@ -175,6 +186,15 @@ define([
 
     _isForceAuthUrl: function () {
       return this.window.location.pathname === '/force_auth';
+    },
+
+    _isSignInUrl: function () {
+      var pathname = this.window.location.pathname;
+      return pathname === '/signin' || pathname === '/oauth/signin';
```

Comment:
If we go with this approach, we should add the information to the docs here: https://github.com/mozilla/fxa-oauth-server/blob/master/docs/api.md